float32 camera_obj_prob #障碍物概率
float32 camera_obj_xmin #相对原点像素偏移
float32 camera_obj_ymin
float32 camera_obj_xmax
float32 camera_obj_ymax
int16 camera_obj_id #在当前帧目标id
int16 camera_obj_class #目标属于种类序号
int16 camera_status #摄像头是否故障